### PR TITLE
feat:(AlertCall): Initial implementation of AlertCall component

### PR DESCRIPTION
**Pull request by jlowery457**
screen shot 2018-05-14 at 10 39 21 am

### CODE REVIEW

File: src/lib/AlertCall/index.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+ * @component alert-call
+ * @variations collab-ui-react
+ */
+
+const AlertCall = props => {
+  const { caller, onAnswerVideo, onAnswerVoice, onReject, show, title } = props;
+
+  const getAvatar = () => {
+    switch(caller.type) {
+      case 'number': return <Avatar title='#' alt={caller.title} size='xlarge'/>;
```
Comment (shashankcs090) [inline src/lib/AlertCall/index.js]: Styling question, should the return be in the next line or in the same line?
@bfbiggs @pauljeter

File: src/lib/AlertCall/index.js [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+  return (
+    show && (
+      <AlertContainer className='cui-alert--call'>
+        <div className='cui-alert__title'>
+          {title}
+        </div>
+        <div className='cui-alert--caller'>
+          {getAvatar()}
+          <div className='cui-alert--caller--title'>
```
Comment (shashankcs090) [inline src/lib/AlertCall/index.js]: Looks like these are elements and not modifiers.
I think these should be cui-alert-caller__title or cui-alert__caller-title, similarly subtitle also.

File: src/lib/AlertCall/index.js [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+import { AlertContainer, Avatar, Button, Icon } from '@collab-ui/react';
+
+/**
+ * @category communication
+ * @component alert-call
+ * @variations collab-ui-react
+ */
+
+const AlertCall = props => {
+  const { caller, onAnswerVideo, onAnswerVoice, onReject, show, title } = props;
```
Comment (shashankcs090) [inline src/lib/AlertCall/index.js]: This looks like a Modal, which occupies the entire screen.
Should the AlertCall component itself have Modal capabilities or should the web-client use this component within an aria-modal?
@pauljeter @bfbiggs

File: src/lib/AlertCall/index.js (hunk 2 above)
Comment (pauljeter) [reply src/lib/AlertCall/index.js]: Agreed.

File: src/lib/AlertCall/index.js (hunk 3 above)
Comment (pauljeter) [reply src/lib/AlertCall/index.js]: It's not exactly like a modal, because you can have 2 on the screen at the same time.

**Pull request merged**